### PR TITLE
iframe replaced by a link for irc

### PR DESCRIPTION
**Pull request by SaptakS**
This solves the issues #21 and #25 . Please review.

### CODE REVIEW

File: pages/index.html [hunk 1]
```diff
@@ -33,9 +33,9 @@
 	<div class="twelve columns">
 		<h1>Connect to #jquery-meeting</h1>
 
-		<p>The jQuery team hosts all of its meeting in the #jquery-meeting IRC channel on Freenode.net. Normally everyone in the channel is muted unless a meeting is going on. Please be respectful to those participating. You can connect to the channel using an IRC client or through the web form below:</p>
+		<p>The jQuery team hosts all of its meeting in the #jquery-meeting IRC channel on Freenode.net. Normally everyone in the channel is muted unless a meeting is going on. Please be respectful to those participating. You can connect to the channel using an IRC client or visit the link below:</p>
```
Comment (scottgonzalez) [inline pages/index.html]: "visit the link below" seems like a poor use of HTML. We should have something like `or use <a ...>webchat</a>`.

**Comment by SaptakS** [comment]
@scottgonzalez  made the changes. Please check.

**Comment by arthurvr** [comment]
Can we fix the trailing whitespace too? :) 

That said the rest looks good to me. @scottgonzalez Are you alright with this? Can you land it? I can't, the content team doesn't have access to this site.

**Comment by scottgonzalez** [comment]
> the content team doesn't have access to this site

Well that seems silly. You now have access :-)

**Comment by arthurvr** [comment]
Ah, thanks. Fixed the whitespace issues myself as it's super trivial. Thanks again!

**Pull request closed without merging**